price-feeder/oracle/provider/provider.go

```diff
@@ -91,3 +91,14 @@ func newCandlePrice(provider, symbol, lastPrice, volume string, timeStamp int64)
 func PastUnixTime(t time.Duration) int64 {
 	return time.Now().Add(t*-1).Unix() * int64(time.Second/time.Millisecond)
 }
+
+// mapPairsToSlice returns the map of currency pairs as slice.
+func mapPairsToSlice(mapPairs map[string]types.CurrencyPair) []types.CurrencyPair {
+	var currencyPairs []types.CurrencyPair
```

Comment:
```suggestion
	currencyPairs := make([]types.CurrencyPair, len(mapPairs))
```

Then you can avoid the `append` and use an index :)

Reply:
This is my bad - I'm used to prioritizing less lines of code, which is not golangish